pkg/exchange/okex/kline_stream.go

```diff
@@ -11,12 +11,15 @@ import (
 type KLineStream struct {
 	types.StandardStream
 
+	exchange types.Exchange
+
 	kLineEventCallbacks []func(candle KLineEvent)
 }
 
-func NewKLineStream() *KLineStream {
+func NewKLineStream(exchange types.Exchange) *KLineStream {
```

Comment:
you don't have to use types.Exchange here, you can use *Exchange since it's all for okx

Reply:
done